### PR TITLE
Add download tests

### PR DESCRIPTION
**Pull request by abnegate**
- Add file download tests

### CODE REVIEW

**Review by Meldiron — APPROVED** [review]

File: tests/Base.php [hunk 1]
```diff
@@ -35,7 +35,7 @@ abstract class Base extends TestCase
         'POST:/v1/mock/tests/general/upload:passed',
     ];
 
-    const EXTENDED_GENERAL_RESPONSES = [
+    const DOWNLOAD_RESPONSES = [
         'Download test passed.',
```
Comment (lohanidamodar) [inline tests/Base.php:39]: @abnegate can we make this response same as others something like `GET:/v1/mock... passed`

File: tests/WebChromiumTest.php [hunk 2]
```diff
@@ -21,6 +21,7 @@ class WebChromiumTest extends Base
         ...Base::BAR_RESPONSES,
         ...Base::GENERAL_RESPONSES,
         ...Base::LARGE_FILE_RESPONSES,
+        "https://appwrite.io/v1/mock/tests/general/download?project=console",
```
Comment (lohanidamodar) [inline tests/WebChromiumTest.php:24]: Why?

**Review by lohanidamodar — CHANGES_REQUESTED** [review]
Looks good, few comments.

File: tests/WebChromiumTest.php (hunk 2 above)
Comment (abnegate) [reply tests/WebChromiumTest.php:24]: @lohanidamodar The Web SDK returns a URL instead of the contents of the URL

**Comment by abnegate** [comment]
@christyjacob4 This one will require an update of the test instance to include this change https://github.com/appwrite/appwrite/pull/3638

**Comment by lohanidamodar** [comment]
@abnegate can we update this with latest changes?

**Comment by lohanidamodar** [comment]
@abnegate Node is failing and also Master has updated Flutter tests that are passing please sync and fix node as well.

**Comment by abnegate** [comment]
@Meldiron Any thoughts on how to fix the CLI here? The endpoint returns a string but it looks like CLI hardcodes the type as `arraybuffer` for `location` routes

**Comment by lohanidamodar** [comment]
@abnegate seems node and deno are still failing.

File: tests/Base.php [hunk 3]
```diff
@@ -41,6 +41,14 @@ abstract class Base extends TestCase
         'GET:/v1/mock/tests/general/download:passed',
     ];
```
Comment (lohanidamodar) [inline tests/Base.php:43]: Let's rename `EXTENDED_GENERAL_RESPONSES` to may be `DOWNLOAD_RESPONSES` ?

**Review by lohanidamodar — CHANGES_REQUESTED** [review]

**Pull request closed without merging**